DXRMapFixups/DeusEx/Classes/DXRFixupM06.uc

```diff
@@ -953,8 +958,21 @@ function AnyEntryMapFixes()
             m.bLocked = false;
             m.bHighlight = true;
         }
+
         HandleJohnSmithDeath();
         FixMaggieMoveSpeed();
+
+        if (dxr.flags.settings.starting_map >= 68 || dxr.flagbase.GetBool('M06_HONGKONG_MJ12LAB_Randomized')) {
```

Comment:
I think Maggie should also be removed if you've been to 06_HONGKONG_STORAGE because of entrance rando you could get to that map first